openapi/index.yaml

```
@@ -394,6 +419,96 @@ definitions:
         format: url
     required:
       - items
+  Group:
+    type: object
+    properties:
+      built_in:
+        type: boolean
+      description:
+        type: string
+      display_name:
+        type: string
+      external_id:
+        type: string
+      group_contract_type:
+        $ref: "#/definitions/GroupType"
+      id:
+        type: string
+      name:
+        type: string
+      type:
+        type: string
+    required:
+      - display_name
```

```suggestion
      display_name:
        type: string
      id:
        type: string
      name:
        type: string
    required:
      - display_name
```
we don't need all the group fields for this API, just return a subset
(you may need to tweak groupContractToApiGroup)